pkg/controller/secret/secret_controller_test.go

```diff
@@ -630,7 +651,7 @@ func Test_ReconcileSecrets(t *testing.T) {
 
 		// Create the secrets for this specific test.
 		if tt.amExists {
-			writeAlertManagerConfig(reconciler, createAlertManagerConfig("", "", ""))
+			writeAlertManagerConfig(reconciler, createAlertManagerConfig("", "", "", "fake-cluster-id"))
```

Comment:
This should also be a blank string (the test confirms that the blank string is overwritten)

Reply:
make sense. Done